tools/metadata.yaml

```diff
@@ -0,0 +1,21 @@
+kind: DroidMetadata
+version: v3
+product: azuresdkforgo
+environments:
+  - name: AZ_SUBSCRIPTION_ID
+    type: secret
+    value: sp.subscriptionid
+  - name: AZ_TENANT_ID
+    type: secret
+    value: sp.tenant
+  - name: AZ_CLIENT_ID
+    type: secret
+    value: sp.clientid
+  - name: AZ_CLIENT_SECRET
+    type: secret
+    value: sp.clientsecret
+  - name: AZ_SP_OBJECT_ID
+    type: secret
+    value: sp.objectid
+  - name: AZ_LOCATION
+    type: arg-mode
```

Comment:
If this is from `--mode`, please change the type to `argument-value-mode`

Source: https://github.com/Azure/adx-automation-agent/blob/ca80c91a2efb7cfcda7eaf62097c8b114f13a256/dispatcher/main.go#L415

Reply:
Fixed :)